adam-core/src/main/scala/org/bdgenomics/adam/models/SingleReadBucket.scala

```diff
@@ -98,9 +62,10 @@ class SingleReadBucketSerializer extends Serializer[SingleReadBucket] {
   }
 
   def write(kryo: Kryo, output: Output, groupedReads: SingleReadBucket) = {
-    writeArray(kryo, output, groupedReads.primaryMapped.toSeq)
-    writeArray(kryo, output, groupedReads.secondaryMapped.toSeq)
-    writeArray(kryo, output, groupedReads.unmapped.toSeq)
+
```

Comment:
I believe we want to drop the SingleReadBucketSerializer and just register this Avro type, right? We don't need to use Kryo for this any more.